enterprise/server/raft/replica/replica.go

```diff
@@ -63,7 +64,7 @@ var (
 type IStore interface {
 	AddRange(rd *rfpb.RangeDescriptor, r *Replica)
 	RemoveRange(rd *rfpb.RangeDescriptor, r *Replica)
-	NotifyUsage(ru *rfpb.ReplicaUsage)
+	NotifyUsage(ru *rfpb.ReplicaUsage, pu []*rfpb.PartitionMetadata)
```

Comment:
could we move []*rfpb.PartitionMetadata inside of ReplicaUsage and leave this API as is?

Reply:
Still in favor of moving this into ReplicaUsage -- because I think we can delete NodeUsage. I'm happy to send a CL now to do so. (StoreUsage is the new NodeUsage, but this data does not need to be in StoreUsage)

Reply:
Moved.